torch_xla/csrc/lowering_context.h

```diff
@@ -18,7 +19,7 @@ namespace ir {
 
 class LoweringContext {
  public:
-  LoweringContext(const std::string& name) : builder_(name) {}
+  explicit LoweringContext(const std::string& name);
```

Comment:
Why not leave this constructor inline and just make it explicit?

Reply:
I had added another ctor which was a bit more involved. At that point I moved both .cpp side.
But then I removed the new one.
So yes, we can move it back .h side.